iris_grib/_save_rules.py

```diff
@@ -1324,6 +1324,24 @@ def product_definition_template_1(cube, grib, full3d_cube=None):
     set_ensemble(cube, grib)
 
 
+def product_definition_template_6(cube, grib, full3d_cube=None):
+    """
+    Set keys within the provided grib message based on Product Definition
+    Template 4.6.
+
+    Template 4.6 is used to represent a percentile forecast at a point in time
+    interval.
+
+    """
+    eccodes.codes_set(grib, "productDefinitionTemplateNumber", 6)
+    if not (cube.coords('percentile') and
```

Comment:
```suggestion
    eccodes.codes_set(grib, "productDefinitionTemplateNumber", 6)
    product_definition_template_common(cube, grib, full3d_cube)
    if not (cube.coords('percentile') and
```

The definition of this template is largely the same as template 1, and I believe should therefore include `product_definition_template_common()`.

https://github.com/SciTools/iris-grib/blob/101feb6478bd3e8eb28f79a1b4b1a844153074ef/iris_grib/_save_rules.py#L1312-L1324